inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -3,6 +3,60 @@
 class S3_Uploads_WP_CLI_Command extends WP_CLI_Command {
 
 	/**
+	 * Verifies the API keys entered will work for writing and deleting from S3.
+	 *
+	 * @subcommand verify
+	 */
+    public function verify_api_keys() {
+
+		S3_Uploads::get_instance(); // Boot
+
+		$upload_dir = wp_upload_dir();
+
+		// The upload file location on the local filesystem
+		$s3_path = $upload_dir['basedir'] . '/' . mt_rand() . '.jpg';
+
+		// Attempt to copy the file to S3
+		WP_CLI::print_value( 'Attempting to upload file '. $s3_path );
+		
+		// Set Error Handler so we can catch it if it fails 
+		set_error_handler( array( $this, 'copy_fail' ) );
+
+		// Copies canolda from the test dir, upto S3
+		$copy = copy(
+			dirname( dirname(__FILE__) ) . '/tests/data/canola.jpg',
+			$s3_path
+		);
+
+		restore_error_handler();
+
+		WP_CLI::print_value( 'File uploaded to S3 successfully' );
+
+		// Delete off S3
+		WP_CLI::print_value( 'Attempting to delete file '. $s3_path );
+		$delete = unlink( $s3_path );
+
+		// Check that delete worked
+		if ( ! $delete ) {
+			WP_CLI::error( 'Failed to delete '. $s3_path );
+			return;
+		}
+
+		WP_CLI::print_value( 'File deleted from S3 successfully' );
+
+		WP_CLI::success( 'Looks like your configuration is correct.' );
+
+    }
+
+    /**
```

Comment:
This is using spaces, not tabs - though we might want to remove, per the comment above.